packages/server/src/automations/tests/utilities/AutomationTestBuilder.ts

```diff
@@ -356,6 +448,16 @@ class AutomationBuilder extends BaseStepBuilder {
     )
   }
 
+  cron(inputs: CronTriggerInputs, outputs?: CronTriggerOutputs) {
```

Comment:
I'm iffy about the ordering here, this is only used in one place at the moment but it feels more natural to have the arguments this way around (the rest of them are reversed).